package.json

```diff
@@ -35,7 +35,7 @@
   "devDependencies": {
     "@types/jquery": "2.0.41",
     "@types/lodash": "4.14.57",
-    "@types/office-js": "0.0.39",
+    "@types/office-js": "^0.0.42",
```

Comment:
For normal libraries, I totally agree, but for a dev-dependency and IntelliSense, I don't see a problem keeping it evergreen.  If we break someone with breaking the d.ts file, we actually _want_ to know.  So for the `@types/office-js` in particular, I would keep it as is.